feed_resources/app/src/main/java/com/firework/example/feedresources/channelhashtags/ChannelHashtagsActivity.kt

```diff
@@ -0,0 +1,85 @@
+package com.firework.example.feedresources.channelhashtags
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.LayoutInflater
+import com.firework.common.feed.FeedResource
+import com.firework.example.feedresources.BuildConfig.FW_CHANNEL_ID
+import com.firework.example.feedresources.R
+import com.firework.example.feedresources.databinding.ActivityChannelHashtagsBinding
+import com.firework.videofeed.FwVideoFeedView
+import com.firework.viewoptions.baseOptions
+import com.firework.viewoptions.viewOptions
+
+class ChannelHashtagsActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityChannelHashtagsBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityChannelHashtagsBinding.inflate(LayoutInflater.from(this))
+        setContentView(binding.root)
+        setTitle(R.string.channel_hashtags_screen_title)
+
+        setupDetails()
+
+        initVideoFeedView()
+
+        binding.etHashtagFilter.setText(HASHTAG_FILTER_EXPRESSION)
+        binding.btnApplyFilter.setOnClickListener {
+            updateViewOptions(binding.etHashtagFilter.text.toString())
+        }
+    }
+
+    @SuppressLint("SetTextI18n")
+    private fun setupDetails() {
+        binding.details.source.text = "Channel"
+        binding.details.channel.text = FW_CHANNEL_ID
+        binding.details.playlist.text = "N/A"
+    }
+
+    private fun initVideoFeedView() {
+        val videoFeedView = binding.fwVideoFeedView
+
+        val viewOptions = viewOptions {
+            baseOptions {
+                feedResource(FeedResource.ChannelHashtag(channelId = FW_CHANNEL_ID, HASHTAG_FILTER_EXPRESSION))
+            }
+        }
+
+        videoFeedView.init(viewOptions)
+    }
+
+    private fun updateViewOptions(hashTagFilter: String) {
```

Comment:
❗ Looks like we do not have a way to update the viewOptions after the initialisation. That's why I had to replace instance of FwVideoFeedView every time new hashtags should be shown.
I created a discussion for this:
https://github.com/loopsocial/firework_sdk_v2/discussions/2020